src/opera/test/pge/cslc_s1/test_cslc_s1_pge.py

```diff
@@ -346,6 +386,72 @@ def test_cslc_s1_output_validation(self):
 
             self.assertIn(f"SAS output file {os.path.basename(expected_output_file)} "
                           f"exists, but is empty", log_contents)
+
+        finally:
+            if os.path.exists(test_runconfig_path):
+                os.unlink(test_runconfig_path)
+
+    def test_geotiff_json_filenames(self):
+        """Test that tiff and json filenames are properly returned."""
+        runconfig_path = join(self.data_dir, 'test_cslc_s1_config.yaml')
+        pge = CslcS1Executor(pge_name="CslcPgeTest", runconfig_path=runconfig_path)
+        pge.run()
+        inner_fname = 'cslc_pge_test/output_dir/t064_135518_iw1/20220501/t064_135518_iw1_20220501_VV.tiff'
+        pge._geotiff_filename(inner_fname)
+        inner_fname = 'cslc_pge_test/output_dir/t064_135518_iw1/20220501/t064_135518_iw1_20220501_VV.json'
+        pge._json_metadata_filename(inner_fname)
+        pge._qa_log_filename()
```

Comment:
I understand that this test is probably providing some additional coverage, but I don't actually see it "testing" anything, so its a little hard to follow. Is there an `assertEquals` that could be added? If the point of the test is just to make sure that an exception is not raised, then we could add a try/except block, where the except block call `self.fail()`, so its clear we do not want it to be hit.